functions/config.php

```diff
@@ -162,6 +162,20 @@ function __init__(){
 			'value'       => $theme_options['btf_link'],
 			'default'	  => '',
 		)),
+		new TextField(array(
+			'name'        => 'UCF Employee Code of Contact link to be used with the image',

```

Comment:
Typo "Code of _Contact_" should be "Code of _Conduct_" on lines 166, 168, 173, and 175